zika/zika.process.py

```diff
@@ -46,5 +46,4 @@
     runner.build_tree()
     runner.timetree_setup_filter_run()
     runner.run_geo_inference()
-    runner.save_as_nexus()
```

Comment:
Is there a reason to not save the Zika tree as a nexus file anymore or was this a deletion for debugging purposes?